.github/actions/build/action.yaml

```diff
@@ -18,7 +18,14 @@ runs:
 
     - name: Install dependencies
       shell: bash
-      run: yarn --prefer-offline --frozen-lockfile
```

Comment:
Wasn't the call to `yarn --prefer-offline --frozen-lockfile` expected to fail already when the `yarn.lock` would changed ?

Reply:
not sure... but trying to avoid the necessity for PRs lik these https://github.com/janus-idp/backstage-plugins/pull/1481

Reply:
yeah ideally `--frozen-lockfile` should have worked but it seems there is an issue with yarn v1 so it doesn't work as expected

https://classic.yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-frozen-lockfile

https://github.com/yarnpkg/yarn/issues/5840#issuecomment-753409393

Reply:
Thanks @invincibleJai !
That makes totally sense now !